packages/editor/src/lib/constants.ts

```diff
@@ -4,7 +4,7 @@ import { EASINGS } from './primitives/easings'
 /** @internal */
 export const DEFAULT_CAMERA_OPTIONS: TLCameraOptions = {
 	isLocked: false,
-	wheelBehavior: 'pan',
+	wheelBehavior: 'auto',
```

Comment:
I changed the default to `auto` because I think that's the most intuitive behavior, but that's of course up to you if you want to do.